client/lib/url/index.js

```diff
@@ -20,4 +21,4 @@ function isExternal( url ) {
 	return isOutsideCalypso( url ) && ! startsWith( url, '//wordpress.com' );
 }
 
-export default { isOutsideCalypso, isExternal };
+export default { isOutsideCalypso, isExternal, support };
```

Comment:
I don't know. We haven't been consistent about what to include in the parent module.